src/pages/api/giftcard-service/new-giftcard.ts

```diff
@@ -0,0 +1,103 @@
+import { NextApiHandler } from "next";
+import { SALEOR_DOMAIN_HEADER } from "@saleor/app-sdk/const";
+import { saleorApp } from "../../../../saleor-app";
+import { createClient } from "../../../lib/graphql";
+import { CreateGiftCardDocument } from "../../../../generated/graphql";
+
+/**
+ * To ensure this endpoint will not be called by unauthorized users,
+ * check secret that should be attached to the request.
+ *
+ * Check external service how secrets are attached
+ *
+ * In this example it will be part of request body.
+ *
+ * In production, put it to ENV
+ */
+const WEBHOOK_SECRET =
+  "jzcccrbnthhjnzsrafwtwcytywsflaokjppljpzzacfhfuwnnshskncxxrkxdqpzxmvypprabiukuojjlhuooajtnovxrieayotqltglpgmsonythblmsmukowrtsfgq";
```

Comment:
Maybe we could put it in the `env.dev` to avoid pattern of hardcoding values in the source code?